ArduPlane/AP_Arming.cpp

```diff
@@ -43,6 +43,12 @@ bool AP_Arming_Plane::pre_arm_checks(bool display_failure)
     // call parent class checks
     bool ret = AP_Arming::pre_arm_checks(display_failure);
 
+    // Check FBWB Max and Min altitude
+    if (plane.g.fbwb_max_altitude_m * 100 < plane.g.FBWB_min_altitude_cm + 1000 && plane.g.fbwb_max_altitude_m > 0) {
+        check_failed(display_failure, "FBWB MAX and MIN altitudes are too close");
```

Comment:
I think its better to name exactly the params and the required difference. Maybe something like this:

ALT_FBW_MAX must be at least 10m above ALT_FBW_MAX

Reply:
Yeah This way it will be easier. Thanks .